### PR TITLE
Video page

### PR DESCRIPTION
**Pull request by juliasugarman**
I killed those errors...finally.

### CODE REVIEW

File: foot-in-the-door/pages/index.js [hunk 1]
```diff
@@ -31,15 +31,10 @@ const events = [
...
-    caption: 'Foot in the Door 5 Artist Video: Ilene Krug Mojsilov',
+    vimeoId: 480895859,
+    caption: 'Foot in the Door 5 Artist: Eyenga Bokamba',
   },
-  {
-    vimeoId: 475231372,
-    caption: 'Foot in the Door 5 Artist: Alonzo Pantoja',
-  },
-]
-
```
Comment (kjell) [inline foot-in-the-door/pages/index.js:42]: Even with all the validation in place, this in invalid javascript that breaks the build? The array needs to be closed with a `]`

File: foot-in-the-door/pages/video.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      <main className="md:flex items-start mb-12">
+        <div className="md:w-1/2 mr-12">
+          <h1 className="text-3xl sm:text-4xl lg:text-5xl font-black tracking-wide">
+            Foot in the Door 5
+          </h1>
+          <h2 className="text-3xl sm:text-4xl lg:text-5xl font-light">
+            Artist Videos
+          </h2>
+          <p className="text-2xl font-light">
+            What makes "Foot in the Door 5: The Virtual Exhibition" so unique?
```
Comment (kjell) [inline foot-in-the-door/pages/video.js:57]: This triggers an eslint error: 

```
react/no-unescaped-entities: `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.`
```

File: foot-in-the-door/pages/video.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            The thousands of participating Minnesota artists! Get to know a few
+            of them in these 1-minute videos. Each artist touches on their
+            creative process and specific artwork featured in the exhibition.
+            Meet participating artists. New videos released each week.
+          </p>
+        </div>
+      </main>
+      <aside className="md:flex pb-6">
+        <div id="video" className="md:w-1/2 pb-8 md: pb-0 md:mr-12">
+          {videos.map((video) => (
```
Comment (kjell) [inline foot-in-the-door/pages/video.js:70]: eslint again. In react, every item in a list needs to have a key: https://reactjs.org/docs/lists-and-keys.html

```
react/jsx-key: Missing "key" prop for element in iterator
```

File: foot-in-the-door/pages/index.js [hunk 4]
```diff
@@ -114,6 +109,11 @@ function Home(props) {
               </a>
             </figcaption>
           </figure>
+          <Link href="/pages/video">
```
Comment (kjell) [inline foot-in-the-door/pages/index.js:112]: `/pages` never goes in the href of a Link. It's the directory the files live in, but never shows to the public. Everything inside of `pages/` becomes `<site URL>/<page name>`, so to make this link work it needs to be `<Link href="/video">`

File: foot-in-the-door/pages/video.js [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+            Foot in the Door 5
+          </h1>
+          <h2 className="text-3xl sm:text-4xl lg:text-5xl font-light">
+            Artist Videos
+          </h2>
+          <p className="text-2xl font-light">
+            What makes "Foot in the Door 5: The Virtual Exhibition" so unique?
+          </p>
+          <p className="text-2xl font-light">
+            {' '}
```
Comment (kjell) [inline foot-in-the-door/pages/video.js:60]: This extra space doesn't do anything, it should be removed

**Review by kjell — CHANGES_REQUESTED** [review]
Found some code errors that need to be fixed

File: foot-in-the-door/pages/video.js [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    vimeoId: 479910635,
+    caption: 'Foot in the Door 5 Artist: Ken Wenzel',
+  },
+  {
+    vimeoId: 478085063,
+    caption: 'Foot in the Door 5 Artist: Paula Warn',
+  },
+]
+
+function video() {
```
[the remark this answers is not in the document]
Comment (juliasugarman) [reply foot-in-the-door/pages/video.js:45]: OH, this wasn't in the Eslint warnings. I can fix this.

File: foot-in-the-door/pages/video.js (hunk 2 above)
Comment (juliasugarman) [reply foot-in-the-door/pages/video.js:57]: I had erased this. I"m not sure why that update wasn't pushed.  I'll push again.

File: foot-in-the-door/pages/video.js (hunk 3 above)
Comment (juliasugarman) [reply foot-in-the-door/pages/video.js:70]: This was also updated. i'm not sure why it wasn't comitted. I'll update all of these.

File: foot-in-the-door/pages/video.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (kjell) [reply foot-in-the-door/pages/video.js:45]: Yes, this is just react knowledge. ESLint might be able to find this problem if the component is referenced in other JSX code, but here it's just exported and ESLint doesn't catch it

**Comment by juliasugarman** [comment]
I'm so sorry that those original updates weren't there. I pushed them again - working on the Meta tags now - but thought you could make sure the other issues were settled.

File: foot-in-the-door/pages/video.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (juliasugarman) [reply foot-in-the-door/pages/video.js]: Should be all set now

**Review by kjell — APPROVED** [review]

**Comment by kjell** [comment]
OK - your changes look good. 

I realized that loading 10+ iframes caused the page to download too slow, so I adjusted it to start with 2, and added a `<details>` that contains the rest of the videos. This way the page loads quickly and users can expand the full list of videos when they are sure they want to watch more. [Preview it here](https://fitd-kjell.lume1.vercel.app/exhibitions/2760/foot-in-the-door/video) and let me know what you think

**Pull request merged**